test/runTest.ts

```diff
@@ -18,21 +18,27 @@ import {
     runTests,
     downloadAndUnzipVSCode,
     resolveCliPathFromVSCodeExecutablePath,
-    ConsoleReporter,
     ProgressReport,
+    ProgressReporter,
+    makeConsoleReporter,
 } from "@vscode/test-electron";
 
-class CIReporter extends ConsoleReporter {
-    constructor(private showProgress: boolean) {
-        super(showProgress);
-    }
+/** ProgressReporter that doesn't show  downloading stage */
+class CIReporter implements ProgressReporter {
```

Comment:
Hi @plemarquand `ConsoleReporter` is no longer a symbol from `test-electron` so I had to update `CIReporter`.